virttest/qemu_storage.py

```diff
@@ -158,6 +158,11 @@ def create(self, params, ignore_errors=False):
                 logging.warning("We'll try to proceed by creating the dir. "
                                 "Other errors may ensue")
                 os.makedirs(image_dirname)
+            elif os.path.exists(self.image_filename):
+                # Remove previous image to avoid qemu-img attempting to get
+                # info as VM might still be running and using it which
+                # causes "write" lock issue.
+                os.unlink(self.image_filename)
```

Comment:
The change would break the support of some operations, such as creating new image on the existed one (includes being or not being used), so can we just apply it for creations in the context of `preprocess_image`?

BTW, I am not sure if qemu could handle this thing well, because the qemu process still holds the file descriptor of the original image, it might has the chance to corrupt the newly created one (not quite sure about this).